src/main/scala/com/databricks/spark/sql/perf/results.scala

```diff
@@ -104,37 +105,87 @@ case class BreakdownResult(
 
 case class Failure(className: String, message: String)
 
-// KEEP ARGUMENTS SORTED BY NAME.
-// It simplifies lookup when checking if a parameter is here already.
-case class MLParams(
+/**
+ * Class wrapping parameters for ML tests; provides the same functionality as MLParams, but is
```

Comment:
Update this text: You are talking about the old MLParams, which won't mean anything to people reading the docs.  That kind of comment belongs in PR comments, not in the code.